src/main/kotlin/com/hj/leetcode/kotlin/problem1662/Solution.kt

```diff
@@ -5,48 +5,18 @@ package com.hj.leetcode.kotlin.problem1662
  */
 class Solution {
     /* Complexity:
-     * Time O(N) and Space O(1) where N is the min flat length of word1 and word2;
```

Comment:
The original time complexity is indeed correct and more accurate. Should revert the change

Reply:
Addressed by https://github.com/hj-core/hj-leetcode-kotlin/pull/194/commits/c5c60f47e7e99b88ce116151a2c533f1d8641638 and https://github.com/hj-core/hj-leetcode-kotlin/pull/194/commits/1870a2a4ab65299e888c3fe0fd4626c412fd3e02